internal/resolution/variablesources/bundle.go

```diff
@@ -119,7 +120,7 @@ func filterBundleDependencies(allBundles []*catalogmetadata.Bundle, bundle *cata
 			catalogfilter.InBlangSemverRange(requiredPackage.SemverRange),
 		))
 		if len(packageDependencyBundles) == 0 {
-			return nil, fmt.Errorf("could not find package dependencies for bundle %q", bundle.Name)
+			return nil, errors.New("could not find package dependencies for bundle")
```

Comment:
Can we add more context to this as well? For example:
```suggestion
			return nil, fmt.Errorf("no bundles found matching required package %q in range %q", requiredPackage.PackageName, requiredPackage.SemverRange)
```